core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -142,7 +141,9 @@ class DagrCoreMain(
   @arg(doc = "Set the memory available to dagr.", common = true)
   val memory: Option[String] = None,
   @arg(doc = "Write an execution report to this file, otherwise write to the stdout", common = true)
-  val report: Option[Path] = None
+  val report: Option[Path] = None,
+  @arg(doc = "Provide an top-like interface for tasks with the give delay in seconds. This suppress info logging.")
+  val taskTop: Double = -1
```

Comment:
Also, do you think it's necessary to make the refresh rate configurable vs. making this just a flag arg?